tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "forceConsistentCasingInFileNames": true,
     "typeRoots": ["./node_modules/@types", "./@types"]
   },
-  "include": ["@types/**/*.ts", "src/**/*.js", "src/**/*.ts", "src/**/*.tsx"]
+  "include": ["@types/**/*.ts", "script/**/*.ts", "src/**/*.js", "src/**/*.ts", "src/**/*.tsx"]
```

Comment:
Can you confirm that we're not accidentally publishing `script/**` type declaration files to the npm package?

Reply:
`component-status.json` won't appear in the package, as it's ephemeral to its own isolated workflow. This can be verified in the [canary release here](https://unpkg.com/browse/@primer/components@0.0.0-2021111183529/dist/). 

@colebemis, the `d.ts` files are [back now too](https://unpkg.com/browse/@primer/components@0.0.0-2021111183529/lib/). Had to add the scripts folder to `tsconfig.build.json`